variants/t-echo/platformio.ini

```diff
@@ -17,6 +17,7 @@ build_flags = ${nrf52840_base.build_flags} -Ivariants/t-echo
   -DEINK_LIMIT_RATE_RESPONSIVE_SEC=1   ; Minimum interval between RESPONSIVE updates
   -DEINK_LIMIT_GHOSTING_PX=2000        ; (Optional) How much image ghosting is tolerated
   -DEINK_BACKGROUND_USES_FAST          ; (Optional) Use FAST refresh for both BACKGROUND and RESPONSIVE, until a limit is reached.
+  -DLFS_NO_ASSERT                      ; Disable LFS assertions
```

Comment:
Since the new code is wrapped in #ifdef ARCH_NRF52 can you add the no-assert to the arch/nrf52.ini definition file instead of just t-echo/platformio.ini ?